ui/app/src/utils/content.ts

```diff
@@ -1126,3 +1126,49 @@ export function getInheritanceParentIdsForField(
   }
   return ids;
 }
+
+export interface GeneratePlaceholderImageDataUrlArgs {
+  width: number;
+  height: number;
+  fillStyle: string;
+  textFillStyle: string;
+  text: string;
+  textPositionX: number;
+  textPositionY: number;
+  font: string;
+  textAlign: CanvasTextAlign;
+}
+
+export function generatePlaceholderImageDataUrl(attributes?: Partial<GeneratePlaceholderImageDataUrlArgs>): string {
+  let attrs: GeneratePlaceholderImageDataUrlArgs = Object.assign(
+    {
+      width: 300,
+      height: 150,
+      fillStyle: '#f0f0f0',
+      text: 'Sample Image',
```

Comment:
Is this parametrized or hard coded to that string?

Reply:
All these are defaults. Whatever comes in attributes prevails.